veps/sig-network/link-state-proposal.md

```diff
@@ -0,0 +1,230 @@
+# VEP #3: Adding Link State Management for vNICs
+
+## Release Signoff Checklist
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+The purpose of this enhancement is to introduce support for link state management (up/down) for virtual NICs in KubeVirt.
+This feature will enable VM owners to dynamically control the link state. It aligns KubeVirt with traditional 
+virtualization platforms and enhances its utility in environments requiring precise network state control.
+
+By introducing link state management, KubeVirt will:
+- Provide VM-level controls to configure the link state of individual interfaces.
```

Comment:
Please be specific on the resource, this sounds like VM specific field rather than VMI.

Reply:
This field appears both in the VM and the VMI.
A user can only edit it on the VM.

Reply:
Let's make it more clear please.

Reply:
How would you phrase it?

Reply:
Maybe mention this in non-goals?